app/api/images/images.controller.js

```diff
@@ -740,7 +695,11 @@ exports.updateAttributes = utils.route(async (req, res) => {
     view_type: req.body.view_type,
     height: req.body.height,
     width: req.body.width,
-    date_orig: req.body.date_orig
+    date_orig: req.body.date_orig,
+    exact_date: exact_date_req,
+    date_shot: req.body.date_shot,
+    date_shot_min: req.body.date_shot_min,
+    date_shot_max: req.body.date_shot_max
```

Comment:
if we don't send req.body.date_shot or date_shot_min, date_shot_max we should probably keep the current one, no?
also we do not have really a way to remove an exact date if we put one by mistake (probably minor issue)

Reply:
True, if date_shot is not sent, exact_date should stay the same. 
Normally that's what happens on line 675/676: check if there is a date_shot in the request (and if null or not), or in the DB (so the current one). 
I think we decided exact_date can't be edited/posted manually, it depends only if there's a date_shot or not.
(That's how I understood it at least)